environment-api/src/main/java/com/sequenceiq/environment/api/v1/environment/endpoint/EnvironmentEndpoint.java

```diff
@@ -184,6 +185,13 @@ SimpleEnvironmentResponses deleteMultipleByCrns(Set<String> crns,
             nickname = "stopEnvironmentByCrnV1")
     void postStopByCrn(@PathParam("crn") String crn);
 
+    @GET
+    @Path("/crn/{crn}/verify")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = EnvironmentOpDescription.VERIFY_BY_CRN, notes = ENVIRONMENT_NOTES,
+            nickname = "verifyEnvironmentByCrn", httpMethod = "GET")
+    CredentialResponse verifyByCrn(@PathParam("crn") String crn);
```

Comment:
pls rename it to `verifyCredentialByCrn` (or `verifyCredentialByEnvCrn`) to make it clear, also pls fix `nickname` in the annotation

Reply:
Yes it makes sense to make it more specific.

Reply:
also maybe it is worth to change the path as well, because now based on path i can think that we are verifying the environment itself